RELEASE_NOTES.md

```diff
@@ -2,9 +2,27 @@ MSA Release Notes
 =================
 
 ### Next
-
-* Added configuration option to Verify Matching Service Test Tool config to allow testing using the new universal dataset
-* Update Test CA certificate
+* [Enabled support for accepting identities from other European countries](https://github.com/alphagov/verify-matching-service-adapter/blob/releaseNotes3.0.0/doc/release-details/3.0.0.md#support-for-european-identities) as part of eIDAS compliance.
+* [Added a configuration option to the Matching Service Test Tool](https://github.com/alphagov/verify-matching-service-adapter/blob/master/verify-matching-service-test-tool/README.md#configuration) to allow testing using the new [universal JSON matching schema](http://alphagov.github.io/rp-onboarding-tech-docs/pages/matching/buildmatchingservice.html#jsonschema).
+* [Upgraded to SHA-256 as the default secure hashing algorithm](https://github.com/alphagov/verify-matching-service-adapter/blob/releaseNotes3.0.0/doc/release-details/3.0.0.md#upgrade-to-sha-256) for signing and encrypting SAML messages.
+* Updated the test IDAP root Certificate Authority.
+* Separated the GOV.UK Verify Hub and IDP truststores to improve security.
+* Included the Hub and IDP truststore files in the JAR archive to simplify the MSA deployment process.
+* Upgraded to using Dropwizard version 1.3.2.
+
+#### Required configuration changes
+* Remove the the `hub.trustStore` section and instead set `environment` in the `metadata` section:
+```diff
+ hub:
+   ssoUrl: https://integration.ida.digital.cabinet-office.gov.uk/SAML2/SSO
+-  trustStore:
+-    path: test_ida_hub.ts
+-    password: puppet
+
+ metadata:
++  # Choices are (INTEGRATION|PRODUCTION)
++  environment: INTEGRATION
+```
```

Comment:
Do we need to mention anything about the eIDAS config here?

Reply:
I don't think so - these are _required_ config changes - your MSA will break unless you make these. You can freely leave out the whole eIDAS section and everything will be fine so it doesn't seem compulsory.

Reply:
That's in the release details for accepting European identities. 
It says what to add to the config file to be able to consume European identities.